Documentation/ceph-csi-drivers.md

```diff
@@ -134,15 +138,15 @@ metadata:
   name: csi-rbd-secret
   namespace: default 
 data:
-  # Key value corresponds to a user name defined in ceph cluster
+  # Key value corresponds to a user name defined in Ceph cluster
   admin: BASE64-ENCODED-PASSWORD
-  # Key value corresponds to a user name defined in ceph cluster
+  # Key value corresponds to a user name defined in Ceph cluster
   kubernetes: BASE64-ENCODED-PASSWORD
   # if monValueFromSecret is set to "monitors", uncomment the
   # following and set the mon there
   #monitors: BASE64-ENCODED-Comma-Delimited-Mons
 ```
-Here, you need your ceph admin/user password encoded in base64. Run `ceph auth ls` in one of your ceph pod, encode the key of your admin/user and replace `BASE64-ENCODED-PASSWORD` by your encoded key.
+Here, you need your Ceph admin/user password encoded in base64. Run `ceph auth ls` in one of your Ceph pod, encode the key of your admin/user and replace `BASE64-ENCODED-PASSWORD` by your encoded key.
```

Comment:
Can you add more steps on how to generate the `kubernetes` user and how to encode the passwords? And why do we need the "kubernetes" user? If we're handing out the admin key, why do we need another user? And can we avoid using the admin key?

Reply:
added